src/main/java/org/gatorgradle/display/CommandOutputSummary.java

```diff
@@ -231,14 +238,34 @@ public void showOutputSummary() {
         isFailure ? "\u001B[1;31m" : "\u001B[1;32m",
         isFailure ? "\u001B[1;35m" : "\u001B[1;32m", log);
 
-    uploadOutputSummary(failed, completedChecks);
+    // Comment out here, for this step is used directly in the report Task
+    // uploadOutputSummary(failed, completedChecks);
```

Comment:
This should be removed, not commented out -- no need to keep things when we can just look back in the Git history, especially if we're not planning on reverting it or changing it much.